### ISSUE TITLE
RTL not supported please fix that

### ISSUE THREAD

**Issue by josephvijayandroiddev** [addisonElliott/SegmentedButton#24]

**Comment by addisonElliott** [comment]
You're right that RTL is not supported for the library. It would take a bit of work to get it working and I likely won't have the time/motivation to complete it.

PRs are welcome though!

**Comment by alex8530** [comment]
check this repo , you will find the solution , and support rtl

https://github.com/Yasilis/SegmentedButton

**Comment by kartikj07** [comment]
It's minSdk is 23. I wanted the same minSdk provided by this project i.e., 16. Do you have any idea for that?
